vermouth/oxDNA/oxDNA.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Provides functionality to read and write oxDNA files.
+"""
+
+from itertools import chain
+
+import numpy as np
+
+from ..molecule import Molecule
+
+
+def read_oxDNA(file_name, exclude=()):
```

Comment:
I would suggest the actual parser doesn't accept a file name, but rather an iterator of lines. This makes reading e.g. network streams or compressed files easier/possible in the future. A wrapper method can open the file.